src/modules/nft/index.js

```diff
@@ -0,0 +1,132 @@
+import Nft from './src';
+export default class NFT {
+  constructor({ network, address, web3, errorHandler }) {
```

Comment:
dont pass `errorHandler` errors should be passed directly to frontend so we can handle it however we decide